lib/matplotlib/patches.py

```diff
@@ -2338,6 +2338,24 @@ def __call__(self, x0, y0, width, height, mutation_size):
             x0, y0 = x0 - pad, y0 - pad
             return Path.circle((x0 + width / 2, y0 + height / 2),
                                max(width, height) / 2)
+    
+    @_register_style(_style_list)
+    class Ellipse:
+        """A ellipse curve."""
+
+        def __init__(self, pad=0.3):
+            """
+            Parameters
+            ----------
+            pad : float, default: 0.3
+                The amount of padding around the original box.
+            """
+            self.pad = pad
+
+        def __call__(self,width, height, mutation_size):
```

Comment:
```suggestion
        def __call__(self, x0, y0, width, height, mutation_size):
```
x0 and y0 are the positions of the text.

You will need to adjust the size, taking pad in to account, for this to work. See the Circle style above.